engine/installation/linux/suse.md

```diff
@@ -149,13 +180,25 @@ the repository.
     $ sudo zypper install docker-ee-<VERSION_STRING>
     ```
 
-    Start Docker:
+4.  Configure Docker to use the `btrfs` filesystem. Edit the file
```

Comment:
ok, would it be worth adding a note that this is only required under certain circumstances?